CHANGELOG.md

```diff
@@ -1,37 +1,113 @@
 # Changelog
 
-## 6.0.0-rc.1
+## 6.0.0
 
-**Note:** This version is a pre-release and is not recommended for production usage.
+Teleport 6.0 is a major release with new features, functionality, and bug fixes.
 
-This release of Teleport contains a number of improvements and bug fixes.
+The major features are [Database Access](https://goteleport.com/teleport/docs/preview/database-access/), open source role-based access control (RBAC), and exposing an easy to use API.
 
-* Added `tsh env` command: [#5395](https://github.com/gravitational/teleport/pull/5395)
-* OSS RBAC: [#5419](https://github.com/gravitational/teleport/pull/5419)
-* Add MySQL support for database access: [#5453](https://github.com/gravitational/teleport/pull/5453)
-* Implement utmp/wtmp support: [#5491](https://github.com/gravitational/teleport/pull/5491)
-* Apply traits to kube, app and database labels: [#5597](https://github.com/gravitational/teleport/pull/5597)
-* Add `kube_public_addr` config field to `proxy_service`:[#5611](https://github.com/gravitational/teleport/pull/5611)
+Users can review the [6.0 milestone](https://github.com/gravitational/teleport/milestone/33?closed=1) on Github for more details.
 
-## 6.0.0-alpha.2
+## New Features
+
+### Database Access
+
+Database Access support was introduced in [RFD #11](https://github.com/gravitational/teleport/blob/master/rfd/0011-database-access.md).
+
+With Database Access users can connect to PostgreSQL and MySQL databases using
+short-lived certificates, configure SSO authentication and role-based access
+controls for databases, and capture SQL query activity in the audit log.
+
+#### Getting Started
+
+Configure Database Access from scratch following the [Getting Started](https://goteleport.com/teleport/docs/database-access/getting-started.mdx)
+guide.
+
+#### Guides
+
+* [AWS RDS/Aurora PostgreSQL](https://goteleport.com/teleport/docs/database-access/guides/postgres-aws.mdx)
```

Comment:
Note: none of these links are valid until we deploy https://github.com/gravitational/teleport/pull/5643.